fabric/integrated_mcp_fabric.py

```
@@ -1,253 +1,246 @@
 
-        self._initialized = connected_count > 0
+        Returns:
+            Optional[Any]: Client object or None if connection failed
+        """
+        try:
+            # Simulate client connection
+            client = {
+                "name": server_name,
+                "url": server_url,
+                "connected": True,
+                "connection_time": datetime.utcnow().isoformat(),
```

Ruff (0.11.9)

`datetime.datetime.utcnow()` used

(DTZ003)